stellar-multitenant/pkg/internal/provisioning/manager_test.go

```diff
@@ -30,126 +36,154 @@ func Test_Manager_ProvisionNewTenant(t *testing.T) {
 	ctx := context.Background()
 
 	messengerClientMock := message.MessengerClientMock{}
-
-	tenantManager := tenant.NewManager(tenant.WithDatabase(dbConnectionPool))
-	p := NewManager(
-		WithDatabase(dbConnectionPool),
-		WithMessengerClient(&messengerClientMock),
-		WithTenantManager(tenantManager),
-	)
-
 	messengerClientMock.
 		On("SendMessage", mock.AnythingOfType("message.Message")).
 		Return(nil)
 
-	t.Run("provision a new tenant for the testnet", func(t *testing.T) {
-		tenantName := "myorg-ukraine"
-		userFirstName := "First"
-		userLastName := "Last"
-		userEmail := "email@email.com"
-		organizationName := "My Org"
-		uiBaseURL := "http://localhost:3000"
-
-		tnt, err := p.ProvisionNewTenant(ctx, tenantName, userFirstName, userLastName, userEmail, organizationName, uiBaseURL, string(utils.TestnetNetworkType))
-		require.NoError(t, err)
+	tenantManager := tenant.NewManager(tenant.WithDatabase(dbConnectionPool))
 
-		schemaName := fmt.Sprintf("sdp_%s", tenantName)
-		assert.Equal(t, tenantName, tnt.Name)
-		assert.Equal(t, uiBaseURL, *tnt.SDPUIBaseURL)
-		assert.Equal(t, tenant.ProvisionedTenantStatus, tnt.Status)
-		assert.True(t, tenant.CheckSchemaExistsFixture(t, ctx, dbConnectionPool, schemaName))
+	pubnetAssets := []string{"USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN", "XLM:"}
+	testnetAssets := []string{"USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5", "XLM:"}
+
+	pubnetWallets := []string{"Freedom Wallet", "Vibrant Assist RC", "Vibrant Assist"}
+	testnetWallets := []string{"Demo Wallet", "Vibrant Assist"}
+
+	user := struct {
+		FirstName string
+		LastName  string
+		Email     string
+		OrgName   string
+	}{
+		FirstName: "First",
+		LastName:  "Last",
+		Email:     "email@email.com",
+		OrgName:   "My Org",
+	}
 
-		// Connecting to the new schema
-		dsn, err := dbConnectionPool.DSN(ctx)
+	assertFixtures := func(tenantName string, isTestnet bool) {
+		tenantDSN, err := router.GetDSNForTenant(dbt.DSN, tenantName)
 		require.NoError(t, err)
-		u, err := url.Parse(dsn)
+
+		tenantDB, err := db.OpenDBConnectionPool(tenantDSN)
 		require.NoError(t, err)
-		uq := u.Query()
-		uq.Set("search_path", schemaName)
-		u.RawQuery = uq.Encode()
+		defer tenantDB.Close()
```

Comment:
💡  It'd be nice to conform to the wording use throughout the project, calling this `{something}dbConnectionPool`.